src/coreComponents/constitutive/solid/PorousSolid.hpp

```diff
@@ -152,6 +155,22 @@ class PorousSolidUpdates : public CoupledSolidUpdates< SOLID_TYPE, BiotPorosity,
     real64 const bulkModulus = m_solidUpdate.getBulkModulus( k );
     real64 const meanEffectiveStressIncrement = bulkModulus * ( strainIncrement[0] + strainIncrement[1] + strainIncrement[2] );
     m_porosityUpdate.updateMeanEffectiveStressIncrement( k, q, meanEffectiveStressIncrement );
+
+    // Compute porosity and its derivatives
+    real64 const deltaPressure = pressure - pressure_n;
+    real64 const deltaTemperature = temperature - temperature_n;
+    real64 porosityInit, dPorosity_dPressure, dPorosity_dTemperature;
+    computePorosity( k,
```

Comment:
@castelletto1 @joshua-white @CusiniM i am not sure if fixed stress version of porosity update should be used here?